models/user.py

```diff
@@ -32,7 +32,10 @@ def has_role(cls, role):
 class UserModel(BaseModel):
     __tablename__ = "users"
 
+    __mapper_args__ = {"polymorphic_identity": "user", "polymorphic_on": "type"}
```

Comment:
I'm thing this should be: thoughts?

```suggestion
    __mapper_args__ = {"polymorphic_identity": "pending_user", "polymorphic_on": "type"}
```

Reply:
To me it makes more sense to keep it as "user".  Inheritance-wise, I tend to think of it as, e.g., "all property managers are also users" rather than "all property managers are also pending users".

Reply:
sounds good! Thanks for the input.